crates/router/tests/health.rs

```diff
@@ -0,0 +1,6 @@
+use router;
```

Comment:
You can remove this as we reference the crate itself

Reply:
removing that won't be able to ref the fn